infra/manifests/billing-service-secondary-sync.dev.yaml

```diff
@@ -15,12 +15,12 @@ spec:
   ports:
     - name: http
```

Comment:
not required

Reply:
fixed

Reply:
Well.. not really - name is still there I just removed the other one